pkg/controller/storageclusterinitialization/storageclusterinitialization_controller.go

```diff
@@ -6,7 +6,7 @@ import (
 
 	"github.com/go-logr/logr"
 	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
-	statusutil "github.com/openshift/ocs-operator/pkg/controller/util"
+	ocssutil "github.com/openshift/ocs-operator/pkg/controller/util"
```

Comment:
Can you also revert this rename? Introduces unnecessary additional diffs.

Reply:
done